src/rpc/rpc_server.cpp

```diff
@@ -121,8 +121,11 @@ grpc::Status CommanderRPCServiceImpl::CreateRandomTx(grpc::ServerContext* contex
     if (!WALLET) {
         reply->set_result("Wallet has not been started");
     } else {
-        WALLET->CreateRandomTx(request->size());
-        reply->set_result("Now wallet is creating tx");
+        if (WALLET->CreateRandomTx(request->size())) {
+            reply->set_result("Now wallet is creating tx");
+        } else {
+            reply->set_result("Wallet is not ready. Did you forget to set passphrase?");
+        }
```

Comment:
I supplement this logic in #137  so I recommend removing it here

Reply:
We can merge it. There’s also changes in Wallet